server/bin/unittests

```diff
@@ -255,6 +259,10 @@ function _setup_state {
     let res=res+$?
     cp -a $_tdir/{job_pool.sh,unittests,pbench*} $_testopt_sat/bin
     let res=res+$?
+    cp -a $_tdir/../../bin/getconf.py $_testopt_com/bin
+    let res=res+$?
+    cp -a $_tdir/../../lib/configtools/__init__.py $_testopt_com/lib/configtools
+    let res=res+$?
```

Comment:
See comments in `bench-scripts/unittests`.

Reply:
ok